config/test/dsa-prototype-ec2.yaml

```diff
@@ -0,0 +1,19 @@
+template_path: "remote/basic-ec2.j2"
+stack_name: "dsa-prototype-ec2"
+dependencies:
+  - "dev/htan-vpc.yaml"
+
+parameters:
+  InstanceType: "i3en.2xlarge"
+  VpcId: !stack_output_external "htan-vpc::VPCId"
+  SubnetId: !stack_output_external "htan-vpc::PrivateSubnet"
+  EncryptVolume: "true"
```

Comment:
default for this is `true` so probably don't need to specify.

Reply:
Yeah, I noticed. Including this just to be explicit/self-documenting.